testnet/README.md

```diff
@@ -27,21 +27,35 @@ Also use `gcloud` to install Kubernetes:
 gcloud components install kubectl
 ```
 
-My GCP project was named "coinkitalpha" and it seemed like Iowa "A" was the best place, so
-I set the `gcloud` defaults with:
+Choose a name for your gcloud coinkit project and create it:
 
 ```
-gcloud config set project coinkitalpha
-gcloud config set compute/zone us-central1-a
+gcloud projects create your_coinkit_project_name
+gcloud config set project your_coinkit_project_name
 ```
 
 It is handy to have `PROJECT_ID` set to the name of your GCP project in your shell,
-so add this to your bash config:
+so add this to your bash config and source it:
 
 ```
 export PROJECT_ID="$(gcloud config get-value project -q)"
 ```
 
+It seemed like Iowa "A" was the best place, so I set the `gcloud` defaults with:
+
+```
+gcloud config set compute/zone us-central1-a
+```
+
+Enable billing for you project: https://cloud.google.com/billing/docs/how-to/modify-project
+
+Add `Kubernetes Engine` and `Container Registry` API access to your project:
```

Comment:
I didn't do this step, is it really needed?

Reply:
It was for me.  I'm guessing that's because the defaults have changed since I made my gcloud account (though I did update gcloud before starting these instructions).  Let me know if you want me to remove.  Doesn't hurt, and could save someone with an older account a few minutes.